spl/src/lib.rs

```diff
@@ -12,6 +12,3 @@ pub mod dex;
 
 #[cfg(feature = "governance")]
 pub mod governance;
-
-#[cfg(feature = "shmem")]
-pub mod shmem;
```

Comment:
Why was `shmem` removed?

Reply:
the change broke the shmem program and armani told me (offline) that instead of fixing it I can just delete it cause it's not used by anyone. In addition, solana will allow for CPIs to return values soon so this program wont be needed anymore.

Reply:
We should remove it.